### DISCUSSION TITLE
Making Money

### DISCUSSION THREAD

**Discussion by Munky25** [someone0002/afterlife-feedback#6] [Ideas]
I know currently there is a large group wanting more good ways to make legitimate money; would the concept of being a lumberjack be a suitable job to add? Ive been to another city with this said system in question, where you go to said location fill up on logs, then go to another spot to mulch them into sawdust, then another location to press them to boards. All locations are at the lumber mill for them they just made it so you could spam e to get 1 log everytime, but I assume it could be setup semi-easily to mimic the style mining is. For example, set up points on trees around the lumber mill you need an axe to use that takes a few seconds, then gives you logs. This would just be a really nice way to add another job that, albeit similar in concept to mining it would add some sort of diversity to the ways we can make money.

**Comment by Munky25** [comment]
Another thought on the aspect of making money making better. Why not add a way to refine gems and make jewelry to sell to the jewelry store that is so frequently robbed.

**Comment by cwalinskid** [comment]
Completely agree, heard we were getting the vineyard job soon. Thing is tho we need legal jobs to pay similar. Like if I put in 3 hours mining I shouldn’t make 10x more than that amount of time delivering cargo. 

My main concern is the recent change in meth price, I have to buy the supply’s, and spend a good amount of time cooking it, which is also a long period of time that I could get locked up. With all that said, it’s not worth it anymore, once you calculate the cost of production and shit you barely make a profit. 

All in all tho loving the server, just think that if people are used to making a lot of money from certain things then you shouldn’t take that away, if anything make the other jobs more lucrative so people are doing something other than mining. Mainly saying this because now I don’t know what to do to make good money in the same amount of time I was.
